packages/connectivity/src/scp-cf/destination/get-subscriber-token.ts

```diff
@@ -49,12 +51,12 @@ export function isSubscriberToken(token: any): token is SubscriberToken {
 export async function getSubscriberToken(
   options: DestinationOptions
 ): Promise<SubscriberToken> {
-  const isXsuaaJwt =
-    !!options.jwt && isXsuaaToken(decodeJwtComplete(options.jwt));
+  const isXsuaaJwt = !!options.jwt && isXsuaaToken(decodeJwt(options.jwt));
+  const userJwt = await retrieveUserToken(options, isXsuaaJwt);
```

Comment:
Why move this to a separate line but leave serviceJwt the same?
Either create separate lines for both or for none of them.

Reply:
Because the userJwt is used to retrieve the serviceJwt.